lib/models/user/requests.js

```diff
@@ -10,12 +10,15 @@ module.exports = function(UserSchema) {
         if (typeof email === "undefined" || email === null || email.length === 0)
             return callback(errors.EMAIL_REQUIRED);
 
+        var requestRejected = this.requested.filter(function(request) {
```

Comment:
Please rename `requestRejected ` to `previousRequestRejected`.

I assumed `requestRejected` was referring to a previous request because I knew what the PR was for, but a later develop won't know this, and it will save them 5-10 minutes of reading code to understand the behavior of this variable in order to know that it applies to previous requests.

Reply:
@mountHouli Agreed and Fixed.